### PR TITLE
What's New in DevTools 123

### PR DESCRIPTION
**Pull request by leahmsft**
Rendered article for review:
* https://review.learn.microsoft.com/microsoft-edge/devtools-guide-chromium/whats-new/2024/03/devtools-123?branch=pr-en-us-3107

AB#49562355

### CODE REVIEW

**Comment by captainbrosset** [comment]
Let's remember to complete https://github.com/MicrosoftDocs/edge-developer/pull/3059 once this PR gets merged.

**Review by mikehoffms — APPROVED** [review]
Approved PR after pushing commits for a Writer/Editor pass.

File: microsoft-edge/devtools-guide-chromium/whats-new/2024/03/devtools-123.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+* [Popover for the selected search result in the Performance > Main track](https://developer.chrome.com/blog/new-in-devtools-123#perf-popover)
+* [Network panel updates](https://developer.chrome.com/blog/new-in-devtools-123#network)
+   * [Clear button and search filter in the Network > EventStream tab](https://developer.chrome.com/blog/new-in-devtools-123#event-stream)
+   * [Tooltips with exemption reasons for third-party cookies in Network > Cookies](https://developer.chrome.com/blog/new-in-devtools-123#3pc-exemption)
+* [Enable and disable all breakpoints in Sources](https://developer.chrome.com/blog/new-in-devtools-123#breakpoints)
+* [View loaded scripts in DevTools for Node.js](https://developer.chrome.com/blog/new-in-devtools-123#node-nav-tree)
+* [Lighthouse 11.5.0](https://developer.chrome.com/blog/new-in-devtools-123#lighthouse)
+* [Accessibility](https://developer.chrome.com/blog/new-in-devtools-123#accessibility)
+* [Miscellaneous highlights](https://developer.chrome.com/blog/new-in-devtools-123#misc)
+<!-- todo: maybe remove some links -->
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/whats-new/2024/03/devtools-123.md:102]: @leahmsft maybe remove some links

File: microsoft-edge/devtools-guide-chromium/whats-new/2024/03/devtools-123.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+For example, to find the largest arrays, you can sort Array​ instances by shallow size, as shown below.
+
+Default behavior for heap snapshots:
+![Heap snapshot default behavior](./devtools-123-images/heap-snapshot-default.png)
+
+Heap snapshots with the new experiment enabled:
+![Heap snapshot with experiment enabled](./devtools-123-images/heap-snapshot-with-experiment.png)
+
+This experiment is helpful because heap snapshots in the Memory tool show a complete list of everything allocated in the V8 and Blink heaps.  Sometimes that list is a little too fine-grained, breaking out an item as if it is multiple items.  For example, the list in the heap snapshot represents each JavaScript Array instance separately from the contents of that Array, which are depicted as _object elements_.<!--todo: link-->  However, in JavaScript code, an Array is a single entity, rather than an array instance and separate contents of the array.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/whats-new/2024/03/devtools-123.md:35]: link object elements
@leahmsft

Comment (leahmsft) [reply microsoft-edge/devtools-guide-chromium/whats-new/2024/03/devtools-123.md:35]: sorry i didn't make it clear that `(object elements)[]` is the name of the object. will fix in next commit

File: microsoft-edge/devtools-guide-chromium/whats-new/2024/03/devtools-123.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+<!-- ====================================================================== -->
+## Memory tool reports backing store sizes as part of an object's shallow size
+
+<!-- Subtitle: New experiment is available to make Shallow Size column in the Memory tool more helpful. -->
+
+Microsoft Edge 123 adds a new experiment called **In heap snapshots, treat backing store size as part of the containing object**.  If you enable this experiment, the **Memory** tool reports shallow sizes in a way that more closely matches JavaScript behavior.  By enabling this experiment, containers and their contents can now be represented as a single item, such as:
+* An Object and its properties.
+* An Array and its contents.
+* A Map and its contents.
+* A Set and its contents.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/whats-new/2024/03/devtools-123.md:25]:
```suggestion
* An `Object` and its properties.
* An `Array` and its contents.
* A `Map` and its contents.
* A `Set` and its contents.
```
Confirmed that they need backticks:
* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object
* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array
* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map
* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set

Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/whats-new/2024/03/devtools-123.md]: added backticks

**Pull request merged**